charts/everest/README.md

```diff
@@ -191,6 +191,6 @@ The following table shows the configurable parameters of the Percona Everest cha
 | server.service.name | string | `"everest"` | Name of the service for everest |
 | server.service.port | int | `8080` | Port to expose on the service. |
 | server.service.type | string | `"ClusterIP"` | Type of service to create. |
-| telemetry | bool | `true` | If set, enabled sending telemetry information. |
+| telemetry | bool | `false` | If set, enabled sending telemetry information. |
```

Comment:
Isn't it a bit confusing that here in the readme we define that the telemetry is false by default, however in the release builds it is actually enabled 🤔
It can be treated as the default readme for the latest dev build though then we are good.

Reply:
Yeah, I agree it can be confusing.. Maybe we can add a comment there saying it will be different in the prod builds